ocs_ci/framework/pytest_customization/reports.py

```diff
@@ -45,5 +45,5 @@ def pytest_sessionfinish(session, exitstatus):
     """
     send email report
     """
-    if ocsci_config.RUN['cli_params']['email']:
+    if ocsci_config.RUN['cli_params'].get('email'):
```

Comment:
This seem like "add-on" commit,  please remove this line.

Reply:
This is required otherwise failing with keyerror

Reply:
weird but we should revisit this one to find out more